### DISCUSSION TITLE
Looking at the PCB, there are 2 pads, one with 5 and another with G.

### DISCUSSION THREAD

**Discussion by clint70** [JosephHewitt/wardriver_rev3#146] [Q&A]
I am looking how to connect up en external power supply (ie. battery or solar).  These 2 pads look suspiciously like 5v and Gnd, but I am not certain.  Before I go and 'blow' something, am I correct on my assumption?

**Comment by JosephHewitt** [comment]
Yes, the `5` and `G` labelled on the back of the board are indeed 5V and ground respectively.
